caravel/sql_lab_utils.py

```diff
@@ -14,7 +14,7 @@ def create_scoped_session():
     engine = create_engine(
         app.config.get('SQLALCHEMY_DATABASE_URI'), convert_unicode=True)
     return scoped_session(sessionmaker(
-        autocommit=True, autoflush=False, bind=engine))
+        autocommit=False, autoflush=False, bind=engine))
```

Comment:
server was complaining about the transactions and wasn't updating the DB.
I've turned it off and commit manually in the sql_lab.py